src/p2p/ModeSystemFuncs.ts

```diff
@@ -670,11 +674,32 @@ export function getExpiredRemovedV3(
     )
   }
 
+  const counters = {
+    add,
+    remove,
+    numApoptosizedRemovals: numApoptosizedRemovals,
+    numProblematicRemovals: numProblematicRemovals,
+    numProblematicNodes: problematicNodes.length,
+    canRemoveProblematicNodesThisCycle: canRemoveProblematicNodesThisCycle,
+    numPotentiallyExpiredRemovals: numPotentiallyExpiredRemovals,
+    numExpiredRemovals: numExpiredRemovals,
+  }
   nestedCountersInstance.countEvent(
-      'p2p',
-      `results of getExpiredRemovedV3: numApoptosizedRemovals: ${numApoptosizedRemovals}, numProblematicRemovals: ${numProblematicRemovals}, numExpiredRemovals: ${numExpiredRemovals}, removed: ${remove}`
+    'p2p',
+    `results of getExpiredRemovedV3: ${JSON.stringify(counters)}`
   )
 
+  const dangerousRemoval = remove === 0 && config.p2p.enableDangerousProblematicNodeRemoval === false
```

Comment:
this variable name is confusing

It should really be 'isSafeRemoval'

why? Because what we are saying here is, unless this dangerousRemove FF is ON, we will return early if its suggested not to remove any nodes.